Assets/HoloToolkit-Preview/QRTracker/README.md

```diff
@@ -5,3 +5,11 @@ QrTracking is implemented in the Windows mixed reality driver. The example scene
 # How to Use?
 There is an example scene �QRCodesSample� this scene shows how to use the QRScanner prefab scripts to talk to the windows mixed reality driver. Whenever a QR code is detected it is uniquely identified and the events are fired which the application can listen to. Added, Updated, and Removed are the events which the apps can expect. The sample scene demonstrates how these events are used, the scene uses the information about the QR code found and displays it in the world. The Id of the QR code is used with the new platform API to get the spatial coordinate system as shown by SpatialGraphCoordinateSystem script. 
 	A second scene �AttachToQRCode� shows how the QR code can be used to attach holograms. As an example, whenever the application sees a QR code with text "pink chair" it places a hologram of the chair near it. The coordinates of the chair relative to the QR code should be adjusted accordingly.
+
+# Enabling/Disabling the feature
+The feature can be turned on/off in the driver using a reg key.
```

Comment:
This seems dangerous 😅

Reply:
How so?

Reply:
Usually registry editing is fairly advanced.

Reply:
Since this is a preview API, the expectation is that people will make a conscious decision to try this feature.

There's also a command provided to flip the reg key on and off, so there won't be any need to dig through the registry itself.

Reply:
No worries, that that is helpful. Not a blocker by any means, but it was curious to see.

Reply:
Agreed! Helpful to clarify use of the API too, since it's currently shipped in a `Preview` namespace.

Reply:
The only thing preventing us from making this a cmd to run is UAC, correct?